react-game/build/asset-manifest.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
Don't store built files in repository. Or put in to branch which never will be to merge with main branch